app/views/merchants/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<head>
+<link rel = 'stylesheet' type = 'text/css' href ='scott_css.css'>
+</head>
+
+<body>
+<div id = "running-header">
+  <h1> amazonia </h1>
+  </div>
+
+<center>
+<h1><%= @merchant.name %></h1>
+<hr>
+<h2>
+  <a href= /merchants/<%= @merchant.id%>/items> All Items</a> | <a href="/merchants/<%= @merchant.id%>/items/new">Add Item</a></h2>
+<p>
+  <%= @merchant.address %><br>
+  <%= @merchant.city %>,
+  <%= @merchant.state %>
+  <%= @merchant.zip %>
+</p>
+  <a href="/merchants/<%= @merchant.id %>/edit">Edit Info</a> | <a href="/merchants">All Merchants</a> |   <a href="/merchants/<%= @merchant.id %>/delete">Delete Merchant</a>
```

Comment:
Looks like your delete link is going to `/merchants/:id/delete`, but your route is defined as `/merchants/:id`, so clicking this link if throwing an error.